project/ibex.json

```diff
@@ -1,26 +1,26 @@
 {
     "srcs": [
-	"src/ibex/ibex_alu.v",
-	"src/ibex/ibex_core.v",
-	"src/ibex/ibex_decoder.v",
-	"src/ibex/ibex_load_store_unit.v",
-	"src/ibex/ibex_fetch_fifo.v",
-	"src/ibex/ibex_id_stage.v",
-	"src/ibex/ibex_if_stage.v",
-	"src/ibex/ibex_pmp.v",
-	"src/ibex/ibex_multdiv_fast.v",
-	"src/ibex/ibex_controller.v",
-	"src/ibex/ibex_compressed_decoder.v",
-	"src/ibex/ibex_core_tracing.v",
-	"src/ibex/ibex_multdiv_slow.v",
-	"src/ibex/ibex_prefetch_buffer.v",
-	"src/ibex/ibex_ex_block.v",
-	"src/ibex/top_artya7.v",
-	"src/ibex/ibex_register_file_ff.v",
-	"src/ibex/clkgen_xil7series.v",
-	"src/ibex/ram_1p.v",
-	"src/ibex/ibex_cs_registers.v"
-        ],
+        "src/ibex/ibex_alu.v",
```

Comment:
Shouldn't this be `third_party` now?

Reply:
We could do it but it won't work for SymbiFlow yet as there were some things to change in the original design.